app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt

```diff
@@ -57,9 +61,22 @@ class DetailsFragment : BottomSheetDialogFragment() {
             else binding.audioMode.setImageResource(R.drawable.mute)
         }
 
+        binding.editFab.setOnClickListener {
+            dismiss()
```

Comment:
reorder some functions, StoreSession.writeLong should be before the code where navigation to newProfileFragment starts, move dismiss() at the end.
Ensure that the app works fine after these changes and in case of any error inform me.